### DISCUSSION TITLE
v4 - Configuring service as Automatic (Delayed start)?

### DISCUSSION THREAD

**Discussion by kcritchnau-qt** [wixtoolset/issues#7786] [Questions]
I have a Windows service that needs to have its startup configured as "Automatic (Delayed Start)." Per the documentation, the supported options for the "Start" enum under Service Install are auto, demand, and disabled. Auto registers the service as "Automatic", and demand registers as "Manual." I know in v3 there was an option for Delayed Start, is there something similar in v4? Thanks in advance!

**Comment by robmen** [comment]
What was the option in v3 that is missing in v4?

**Reply by vlaterz** [reply]
I have the same question.

https://wixtoolset.org/docs/v3/xsd/wix/serviceconfig/
In WIX3 it is said that ServiceConfig has DelayedAutoStart parameter.

But when i try to use such parameter in my WIX4 project, it says "ServiceConfig element contains an unexpected attribute 'DelayedAutoStart' "

**Reply by robmen** [reply]
Take a look at the code. If you find something is missing, definitely open an issue.
